### ISSUE TITLE
Link to both media.ccc.de and Youtube?

### ISSUE THREAD

**Issue by killercup** [Rustaceans/rust-cologne#36]
A few of our talks have videos, and so far they are available at both media.ccc.de and Youtube. Do we always want to link to one of them, or link to both?

The media.ccc.de version is usually available earlier (shouldn't matter) and is free from user tracking and ads. Youtube has more device support (and mobile-friendly quality settings) as well as suggestions for more Rust videos from the official Rust channel.

This might require some minor tweaks to the website.

(I though of this after seeing the positive reactions the Youtube link of @Florob's latest talk got [on /r/rust](https://www.reddit.com/r/rust/comments/6nj046/rust_cologne_june_florian_zeitz_inline_assembly/).)

**Comment by colin-kiegel** [comment]
My personal preference would be to link to youtube (or both).

But on the other hand, @Florob does all the video editing work. So I would also be ok, to make this his pick.

**Comment by Florob** [comment]
From what I can see the feedback was about the talk in general, and not the link being to YouTube though, was it?
From my point of view, we publish on both, because they each have their own set of benefits. So it probably makes sense to link to both, too.
In this particular case the media.ccc.de video was available more than a month earlier, so I'm not entirely sure "shouldn't matter" is a fair characterization in general.

**Comment by killercup** [comment]
Okay, let's link to both, then!

- - -

> From what I can see the feedback was about the talk in general, and not the link being to YouTube though, was it?


Right, we could have posted the media.ccc.de link and got the same reactions; they only referred to the (great) content. The video getting uploaded on Youtube was just the occasion to post it to Reddit. (Which in turn caused me see it and open this issue.)

(Related: It might be a good idea to post links to recorded meetups to Reddit/comment on the Meetup event as they become available, to inform folks who couldn't attend or missed the initial announcement.)

> In this particular case the media.ccc.de video was available more than a month earlier, so I'm not entirely sure "shouldn't matter" is a fair characterization in general.

In this case there was a month in between, but: If we wanted to, we could upload it to Youtube at basically the same time. Jan-Erik has access to the Rust channel's account. That's what the "shouldn't matter" refers to :)

**Comment by badboy** [comment]
Yeah, I completely forgot to upload it to the channel. Next time I try to be faster on that. :) 
I'd say we should link to both.
